src/style.css

```diff
@@ -1 +1,70 @@
-/* Styles go here */
+body {
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  margin: 0;
+}
+
+[class*="__link"] {
+  text-decoration: none;
+  color: inherit;
+  cursor: pointer;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  background: #fff;
+}
+
+.logo__link {
+  height: 40px;
+}
+
+.logo__img {
+  height: 100%;
+  width: 100%;
+}
+
+.nav {
+  display: flex;
+}
+
+.nav__item:not(:first-child) {
+  margin: 0 0 0 20px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__link {
+  position: relative;
+  text-transform: uppercase;
+  display: flex;
+  white-space: nowrap;
+  align-items: center;
+  height: 60px;
+  transition: all ease 0.3s;
+}
+
+.nav__link:hover,
+.nav__link.is-active {
+  color: #00acdc;
+}
```

Comment:
don't group selectors together
Later you can add some other grouping 
and finally will need to check many places to get styles for 1 selector